webpack.requirejs_jquery_q.config.js

```diff
@@ -0,0 +1,25 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+  entry: './lib/index.js',
+  output: {
+    path: path.join(__dirname, 'dist', 'browser'),
+    filename: 'OktaAuthRequireJquery.js',
+    libraryTarget: 'amd'
+  },
+  externals: [
+    'jquery',
+    'q'
+  ],
+  resolve: {
+    alias: {
+      'ajaxRequest': './jqueryRequest'
+    }
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      SDK_VERSION: JSON.stringify(require('./package.json').version)
```

Comment:
I think it's odd that this needs to be stringified, but it's the only way webpack will insert the version as a string in quotes, rather than a raw string.